wifi/.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
```

Comment:
Oops, from this point onwards, looks like you committed the wifi repo within eWine-connectivity-manager repo. Revert this please.